### DISCUSSION TITLE
话说现在为啥没多少人改ltw3，服务器有人来吗

### DISCUSSION THREAD

**Discussion by misividkoukou** [Meloong-Git/PCL#2213] [讨论]
挺喜欢ltw的，我们打算自己再做一个版本，初步计划增加23个小游戏与多个新道具，开发周期5年，不知道有没有想来的（
再来找找有没有玩服务器的？1.20生电，开了两年半，找点有技术的，摸鱼的

**Comment by iPodtouch6557** [comment]
我自己开了一个呢（）也是1.20.1生电，懒得自己搞装的庄主做的，主城快做好了

**Comment by SALTWOOD** [comment]
我开了一个服务器，1.20.1半生存
